### PR TITLE
Add "Start from template" step under CLI -> Get Started

### PR DESCRIPTION
**Pull request by igor-p**
### Description

Adds a section under`CLI` -> `Get Started` called `Start from template` that demonstrates how to set up a Move package from a template. Uses this command:
```bash
aptos move init --name hello_blockchain --template hello-blockchain
```

image

image

### Checklist

- Do all Lints pass?
  - [ ] Have you ran `pnpm spellcheck`?
  - [x] Have you ran `pnpm fmt`?
  - [x] Have you ran `pnpm lint`?

### CODE REVIEW

File: apps/nextra/pages/en/build/cli/start-from-template.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <Card href="../smart-contracts">
+      <Card.Title>Smart Contracts</Card.Title>
+      <Card.Description>Learn how to build in Move</Card.Description>
+    </Card>
+    <Card href="../smart-contracts/create-package">
+      <Card.Title>Create Package</Card.Title>
+      <Card.Description>Get started by learning how to create a Move package</Card.Description>
+    </Card>
+  </Cards>
+
```
Comment (hariria) [inline apps/nextra/pages/en/build/cli/start-from-template.mdx:43]: Nit remove

File: apps/nextra/pages/en/build/cli/start-from-template.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { Card, Cards, RemoteCodeblock, permalinkFetch } from '@components/index';
+import { Steps } from "nextra/components"
+
+export async function getStaticProps() {
+  return await permalinkFetch([
+    "https://github.com/aptos-labs/aptos-core/blob/77e1d222ebc5e7294e115e0d090c001da1d0e072/aptos-move/move-examples/hello_blockchain/sources/hello_blockchain.move#L1-L59"
```
Comment (hariria) [inline apps/nextra/pages/en/build/cli/start-from-template.mdx:6]: You're going to want this one actually: https://github.com/aptos-labs/aptos-core/blob/afd3706c17bcccfb39a9d6059aecbfa648ed295d/aptos-move/move-examples/hello_blockchain/sources/hello_blockchain.move#L4

**Review by hariria — COMMENTED** [review]
Nice start. Could you also add a tab perhaps to https://developer-docs-nextra-git-igor-cli-template-aptoslabs.vercel.app/en/build/smart-contracts/create-package or a link to this page from there so that people navigating to smart contracts can also see how to use the templates?

**Comment by igor-p** [comment]
> Nice start. Could you also add a tab perhaps to https://developer-docs-nextra-git-igor-cli-template-aptoslabs.vercel.app/en/build/smart-contracts/create-package or a link to this page from there so that people navigating to smart contracts can also see how to use the templates?

Good point. Do you think a callout would be ideal? Something like this:

image

**Review by hariria — APPROVED** [review]

**Review by gregnazario — APPROVED** [review]

**Comment by hariria** [comment]
@igor-p feel free to merge these in when you're ready

**Pull request merged**